packages/stockflux-components/src/index.css

```diff
@@ -61,7 +61,7 @@
 
   /* padding */
   --padding-small: 5px;
-  --padding-medium: 10px;
+  --padding-medium: 12px;
```

Comment:
Does this not break any other components?

Reply:
I have checked through the other references where this is used, and it makes no difference to them.
- One is for when the mini-chart errors. so the change just aligns the text to better fit the section
- The other is in the padding of watch list items to the left, so now it matches the scrollbar width making it centred.